app/Models/User.php

```diff
@@ -49,6 +49,15 @@ class User extends Authenticatable implements MustVerifyEmail, HasLocalePreferen
 
     public const DISTANCE_UNIT_KM = 'km';
 
+    /**
+     * Possible contact sort order.
+     */
+    public const CONTACT_SORT_ORDER_ASC = 'a_to_z';
+
+    public const CONTACT_SORT_ORDER_DESC = 'z_to_a';
```

Comment:
I would prefer calling it:
```suggestion
    public const CONTACT_SORT_ORDER_ASC = 'asc';

    public const CONTACT_SORT_ORDER_DESC = 'desc';
```

Reply:
doh. yeah you are right.